src/modules/google-organization-kp/includes/Rest_Controller.php

```diff
@@ -0,0 +1,354 @@
+			"OM" => "Oman",
+			"PK" => "Pakistan",
+			"PW" => "Palau",
+			"PS" => "Palestinian Territory, Occupied",
+			"PA" => "Panama",
+			"PG" => "Papua New Guinea",
+			"PY" => "Paraguay",
+			"PE" => "Peru",
+			"PH" => "Philippines",
+			"PN" => "Pitcairn",
+			"PL" => "Poland",
+			"PT" => "Portugal",
+			"PR" => "Puerto Rico",
+			"QA" => "Qatar",
+			"RE" => "Reunion",
+			"RO" => "Romania",
+			"RU" => "Russian Federation",
+			"RW" => "Rwanda",
+			"SH" => "Saint Helena",
+			"KN" => "Saint Kitts and Nevis",
+			"LC" => "Saint Lucia",
+			"PM" => "Saint Pierre and Miquelon",
+			"VC" => "Saint Vincent and the Grenadines",
+			"WS" => "Samoa",
+			"SM" => "San Marino",
+			"ST" => "Sao Tome and Principe",
+			"SA" => "Saudi Arabia",
+			"SN" => "Senegal",
+			"CS" => "Serbia and Montenegro",
+			"SC" => "Seychelles",
+			"SL" => "Sierra Leone",
+			"SG" => "Singapore",
+			"SK" => "Slovakia",
+			"SI" => "Slovenia",
+			"SB" => "Solomon Islands",
+			"SO" => "Somalia",
+			"ZA" => "South Africa",
+			"GS" => "South Georgia and the South Sandwich Islands",
+			"ES" => "Spain",
+			"LK" => "Sri Lanka",
+			"SD" => "Sudan",
+			"SR" => "Suriname",
+			"SJ" => "Svalbard and Jan Mayen",
+			"SZ" => "Swaziland",
+			"SE" => "Sweden",
+			"CH" => "Switzerland",
+			"SY" => "Syrian Arab Republic",
+			"TW" => "Taiwan, Province of China",
+			"TJ" => "Tajikistan",
+			"TZ" => "Tanzania, United Republic of",
+			"TH" => "Thailand",
+			"TL" => "Timor-Leste",
+			"TG" => "Togo",
+			"TK" => "Tokelau",
+			"TO" => "Tonga",
+			"TT" => "Trinidad and Tobago",
+			"TN" => "Tunisia",
+			"TR" => "Turkey",
+			"TM" => "Turkmenistan",
+			"TC" => "Turks and Caicos Islands",
+			"TV" => "Tuvalu",
+			"UG" => "Uganda",
+			"UA" => "Ukraine",
+			"AE" => "United Arab Emirates",
+			"GB" => "United Kingdom",
+			"US" => "United States",
+			"UM" => "United States Minor Outlying Islands",
+			"UY" => "Uruguay",
+			"UZ" => "Uzbekistan",
+			"VU" => "Vanuatu",
+			"VE" => "Venezuela",
+			"VN" => "Viet Nam",
+			"VG" => "Virgin Islands, British",
+			"VI" => "Virgin Islands, U.s.",
+			"WF" => "Wallis and Futuna",
+			"EH" => "Western Sahara",
+			"YE" => "Yemen",
+			"ZM" => "Zambia",
+			"ZW" => "Zimbabwe"
+		);
+		
+		$data = array();
+		foreach( $countries as $country_code => $country_name ) {
+			$data[] = array(
+				'code' => $country_code,
+				'name' => $country_name
+			);
+		}
+
+		return rest_ensure_response( $data );
+	}
+
+	public function existing_data() {
+		$configuration_service = Wordlift_Configuration_Service::get_instance();
+		$publisher_id = $configuration_service->get_publisher_id();
+
+		if ( empty($publisher_id) || $publisher_id == "(none)" ) {
+			return new \WP_Error(
+				'wl_google_organization_kp_data_not_found',
+				__( 'No data was found.', 'wordlift' ),
+				array(
+					'status' => 404,
+				)
+			);
+		}
+
+		// @todo: Use publisher_id to retrieve data and return it.
+
+		return rest_ensure_response( $publisher_id );
```

Comment:
Here we need to expand this response and provide all the fields that you expect in the UI. You'll need some help to understand where are all the fields map, but you can start with post_title, entity_type, etc.

Reply:
I'll start with these fields and when I'm ready for help with the rest I will let you know.